src/server/game/Server/Packets/MiscPackets.cpp

```diff
@@ -711,3 +711,32 @@ WorldPacket const* WorldPackets::Misc::StartTimer::Write()
 
     return &_worldPacket;
 }
+
+void WorldPackets::Misc::AdventureJournalOpenQuest::Read()
+{
+    _worldPacket >> AdventureJournalID;
```

Comment:
uses casts like you did with .size()

e.g.
```c
_worldPacket << uint32(AdventureJournalID);
```
in this case

do this for all primitive datatypes